lib/rules/sort-invocations.js

```diff
@@ -0,0 +1,305 @@
+import { builders as b } from 'ember-template-recast';
+
+import Rule from './_base.js';
+
+function cloneIfTextNode(node) {
+  if (node.type === 'MustacheStatement') {
+    return node;
+  }
+
+  return b.text(node.chars);
+}
+
+function getAttributeName(node) {
+  return node.name;
+}
+
+function getAttributePosition(node) {
+  const name = getAttributeName(node);
+
+  if (name.startsWith('@')) {
+    return 1;
+  }
+
+  if (name === '...attributes') {
```

Comment:
wondering if we need specific order for `data-test-` attributes

Reply:
I don't think so. `data-test-` attributes, used for testing, are just a subset of [(data) attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/How_to/Use_data_attributes). We can let them be alphabetized as usual.